blazor/rich-text-editor/data-binding.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Value Binding in Blazor RichTextEditor
```

Comment:
Binding  - is it the correct casing?

Reply:
In given document outline mentioned like this sir.

<img width="559" alt="image" src="https://user-images.githubusercontent.com/93525217/201014928-201a5381-0513-46ab-a357-8a5290b40a45.png">

Reply:
In other team components are mentioned like "Value Biding " only sir